bin/check-versions.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+INSTALLED_VERSION=`node -p -e "require('prom-client/package.json').version"`
+LATEST_VERSION=`npm show prom-client version`
+
+if [[ "$INSTALLED_VERSION" != "$LATEST_VERSION" ]]; then
+	echo "You currently have prom-client@${INSTALLED_VERSION} in devDependencies, but you need to have prom-client@${LATEST_VERSION}.";
+	echo 'Please run `npm install --save-dev --force prom-client`';
+	exit 1
+fi
```

Comment:
FreeBSD doesn't have bash by default, and Windows basically never does. In another package I help maintain, we switched our bash script to a node.js script to avoid compat issues. (See https://github.com/Automattic/node-canvas/pull/754.)

Reply:
:+1: I was thinking about potentially doing the install at benchmark runtime instead? What do you think? Would avoid the nasty `--force` you have to run to install yourself as a devDep, and would let us run technically against multiple versions (I would install into a tmp directory and require from there). Thoughts?

Reply:
But at the very least, 👍 to moving this check into nodejs.

Reply:
I extracted the benchmark runner, it now installs the requested version(s) on demand.

Reply:
I will need to fix up my newly extracted module to be `node@6` compatible, but I will do that next. Hopefully this looks like a good API for the package. I should have it ready and fully tested on node@6 on Monday.

Reply:
there is https://www.npmjs.com/package/libnpm now, btw